src/middlewares/checkAuth.js

```diff
@@ -6,23 +6,25 @@ import { deleteToken, setToken, getToken } from '../utils/helpers/initRedis';
 const { Users } = models;
 
 const checkAuth = async (req, res, next) => {
-  
   try {
-    let token;
-    if (req.headers.jwt) {
-      token = req.headers.jwt;
+    let token = '';
+    if (req.headers.token) {
+      token = req.headers.token;
```

Comment:
why not just send it with the header JWT from the frontend ?

Reply:
Yeah, we are sending the token with the header JWT from the frontend, and using the BE code you have highlighted to receive the token from FE.